config/crd/bases/anywhere.eks.amazonaws.com_cloudstackmachineconfigs.yaml

```diff
@@ -160,6 +160,16 @@ spec:
           status:
             description: CloudStackMachineConfigStatus defines the observed state
               of CloudStackMachineConfig
+            properties:
+              failureMessage:
+                description: FailureMessage indicates that there is a fatal problem
+                  reconciling the state, and will be set to a descriptive error message.
+                type: string
+              specValid:
+                description: 'Important: Run "make" to regenerate code after modifying
```

Comment:
I think this might need to be regenerated?

Reply:
Added spacing and regenerated, thanks